src/core/include/openvino/op/reverse_sequence.hpp

```diff
@@ -38,24 +38,26 @@ class OPENVINO_API ReverseSequence : public Op {
     int64_t get_origin_batch_axis() const {
         return m_batch_axis;
     }
-    void set_batch_axis(int64_t batch_axis) {
-        m_batch_axis = batch_axis;
-    }
+    void set_batch_axis(int64_t batch_axis);
```

Comment:
Please don't change setter functions

Reply:
In doing this, I mean to skip axis normalization in `shape_infer` function, otherwise, `shape_infer` should normalize axis to avoid the effect from those setter functions.

Reply:
I agree with you. Let's change setters to avoid axes normalization at the shape_infer stage